packages/lib-react-components/src/Markdownz/Markdownz.js

```diff
@@ -111,26 +111,32 @@ class Markdownz extends React.Component {
     const remarkReactComponents = Object.assign({}, componentMappings, components)
     const remarkSettings = Object.assign({}, settings)
 
-    const markdown = remark()
-      .data('settings', remarkSettings)
-      .use(emoji)
-      .use(remarkSubSuper)
-      .use(externalLinks)
-      .use(footnotes, { inlineNotes: true })
-      .use(ping, {
-        ping: (resource, symbol) => this.shouldResourceBeLinkable(resource, symbol), // We could support passing in a prop to call a function here
-        pingSymbols: [at, hashtag, subjectSymbol],
-        resourceURL: (resource, symbol) => this.buildResourceURL(resource, symbol)
-      })
-      .use(toc)
-      .use(remark2react, { remarkReactComponents })
-      .processSync(newChildren).result
-
-    return (
-      <React.Fragment>
-        {markdown}
-      </React.Fragment>
-    )
+    try {
+      const markdown = remark()
+        .data('settings', remarkSettings)
+        .use(emoji)
+        .use(remarkSubSuper)
+        .use(externalLinks)
+        .use(footnotes, { inlineNotes: true })
+        .use(ping, {
+          ping: (resource, symbol) => this.shouldResourceBeLinkable(resource, symbol), // We could support passing in a prop to call a function here
+          pingSymbols: [at, hashtag, subjectSymbol],
+          resourceURL: (resource, symbol) => this.buildResourceURL(resource, symbol)
+        })
+        .use(toc)
+        .use(remark2react, { remarkReactComponents })
+        .processSync(newChildren).result
+
+      return (
+        <React.Fragment>
+          {markdown}
+        </React.Fragment>
+      )
+    } catch (error) {
+      console.error(error)
+      return <p>{error.message}</p>
```

Comment:
[rambling to self] I'm trying to imagine situations where you might not want to render a `<p>` to display an error message... but in the context of app-project and lib-classifier, I can't think of any. This is fine. 👌